command-snapshot.json

```diff
@@ -34,7 +34,22 @@
   {
     "command": "force:source:deploy:report",
     "plugin": "@salesforce/plugin-source",
-    "flags": ["apiversion", "jobid", "json", "loglevel", "targetusername", "wait", "verbose"]
+    "flags": ["apiversion", "jobid", "json", "loglevel", "targetusername", "verbose", "wait"]
+  },
+  {
+    "command": "force:source:manifest:create",
+    "plugin": "@salesforce/plugin-source",
+    "flags": [
+      "apiversion",
+      "json",
+      "loglevel",
+      "manifestname",
+      "manifesttype",
+      "metadata",
+      "outputdir",
+      "sourcepath",
+      "targetusername"
```

Comment:
@shetzel to use `SfdxCommand`'s standardized `--apiversion` flag I had to set `supportsUsername=true` which also includes the `--targetusername`... Do you think it would be better to duplicate the `apiversion` flag?

Reply:
Unless the command will use that flag (-u) I would not set `supportsUsername=true`.  Just duplicate the apiversion flag.  There might be a way to reference it without duplicating.